### DISCUSSION TITLE
Mediafinder Copy folder / Copy & move image

### DISCUSSION THREAD

**Discussion by AIC-BV** [wintercms/winter#1048] [Ideas]
Ideally, when working with multiple languages, you should have an image for each language.
For example:
- actie.jpg (NL)
- action.jpg (EN & FR)
- aktion.jpg (DE)
- ...
Even though the image might be exactly the same, for SEO purposes, it should have its name in the correct language.

To make this job easier, being able to copy an image through the mediafinder (and give it a new name) can drastically shorten the duration of this boring task

**Comment by LukeTowers** [comment]
@AIC-BV I'm definitely open to having a PR submitted 😉

**Comment by damsfx** [comment]
@AIC-BV see: https://github.com/wintercms/winter/pull/1083
